app/lib/notifications.server.ts

```diff
@@ -34,7 +34,7 @@ const sendNotification = async (emailAddress: string, subject: string, text: str
   try {
```

Comment:
You can remove the `async` from this function since you are no longer using await